src/gui/socketapi.cpp

```diff
@@ -366,8 +396,20 @@ void SocketApi::broadcastMessage(const QString &msg, bool doWait)
     }
 }
 
-void SocketApi::processShareRequest(const QString &localFile, SocketListener *listener, ShareDialogStartPage startPage)
+void SocketApi::processShareRequest(const QString &localFileC, SocketListener *listener, ShareDialogStartPage startPage)
 {
+    QString localFile = localFileC;
+
+#if defined(Q_OS_WIN)
+    OCC::ConfigFile Cfg;
+    char letter[2];
```

Comment:
It is wrapped back into a QString later on, so why not just get the first QChar of that string?